torch/csrc/jit/script/compiler.h

```diff
@@ -122,8 +122,25 @@ struct TORCH_API BuiltinFunction : public SugaredValue {
       size_t n_binders) override;
 };
 
+struct TORCH_API BuiltinModule : public SugaredValue {
+  BuiltinModule(const std::string& name)
+    : name(name) {}
+  std::string name;
+
+  virtual std::string kind() const override {
```

Comment:
`virtual` + `override` is redundant and our linters will complain once I activate them, please only use `override`. I frequently have to codemod the repo to undo this